llvm/lib/Passes/PassBuilder.cpp

```diff
@@ -231,6 +231,7 @@
 #include "llvm/Transforms/Scalar/TLSVariableHoist.h"
 #include "llvm/Transforms/Scalar/TailRecursionElimination.h"
 #include "llvm/Transforms/Scalar/WarnMissedTransforms.h"
+#include "llvm/Transforms/SPMDTransforms/Passes.h"
```

Comment:
This line needs to be moved before `#include "llvm/Transforms/Scalar/ADCE.h"` -- clang-format would put **Capitalized** entries before **Uncapitalized** ones.

Reply:
ok, I will update it.  I did clang-format before submit. It's strange.